jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonMarshaller.java

```diff
@@ -119,1016 +234,1593 @@ public String marshall(Definitions def, String preProcessingData) throws IOExcep
                 } else if (rootElement instanceof Escalation) {
                     // TODO
                 } else if (rootElement instanceof Collaboration) {
 
                 } else {
-                	_logger.warn("Unknown root element " + rootElement + ". This element will not be parsed.");
-	            }
-	        }
-
-	        generator.writeObjectFieldStart("stencilset");
-	        generator.writeObjectField("url", this.profile.getStencilSetURL());
-	        generator.writeObjectField("namespace", this.profile.getStencilSetNamespaceURL());
-	        generator.writeEndObject();
-	        generator.writeArrayFieldStart("ssextensions");
-	        generator.writeObject(this.profile.getStencilSetExtensionURL());
-	        generator.writeEndArray();
-	        generator.writeEndObject();
+                    _logger.warn("Unknown root element " + rootElement + ". This element will not be parsed.");
+                }
+            }
+
+            generator.writeObjectFieldStart("stencilset");
+            generator.writeObjectField("url",
+                                       this.profile.getStencilSetURL());
+            generator.writeObjectField("namespace",
+                                       this.profile.getStencilSetNamespaceURL());
+            generator.writeEndObject();
+            generator.writeArrayFieldStart("ssextensions");
+            generator.writeObject(this.profile.getStencilSetExtensionURL());
+            generator.writeEndArray();
+            generator.writeEndObject();
         } finally {
-        	_diagramElements.clear();
+            _diagramElements.clear();
         }
     }
 
-
-    /** protected void marshallMessage(Message message, Definitions def, JsonGenerator generator) throws JsonGenerationException, IOException {
-        Map<String, Object> properties = new LinkedHashMap<String, Object>();
-
-        generator.writeStartObject();
-        generator.writeObjectField("resourceId", message.getId());
-
-        properties.put("name", message.getName());
-        if(message.getDocumentation() != null && message.getDocumentation().size() > 0) {
-            properties.put("documentation", message.getDocumentation().get(0).getText());
-        }
-
-        marshallProperties(properties, generator);
-        generator.writeObjectFieldStart("stencil");
-        generator.writeObjectField("id", "Message");
-        generator.writeEndObject();
-        generator.writeArrayFieldStart("childShapes");
-        generator.writeEndArray();
-        generator.writeArrayFieldStart("outgoing");
-        generator.writeEndArray();
-
-        generator.writeEndObject();
-    } **/
-
-    protected void marshallCallableElement(CallableElement callableElement, Definitions def, JsonGenerator generator) throws JsonGenerationException, IOException {
+    /**
+     * protected void marshallMessage(Message message, Definitions def, JsonGenerator generator) throws JsonGenerationException, IOException {
```

Comment:
@tsurdilo probably worth to delete completely this `marshallMessage` if it is not used.